bower.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sendgrid2sparkpost",
+  "description": "",
+  "main": "index.js",
+  "authors": [
+    "Mohammad Hossain <mohammad.hossain@sparkpost.com>"
```

Comment:
npm didn't support multiple author(s), so overlooked it's an array. i'll add in both places :).